e2e/specs/st_write.spec.js

```diff
@@ -39,12 +39,6 @@ describe("st.write", () => {
     cy.get(".stDataFrame").should("have.length", 1);
   });
 
-  it("st.write should display pyspark.sql.DataFrame as st.dataframe with proper width and height", () => {
```

Comment:
Should this be getting refactored or just plain out deleted?

Reply:
There is another test in the e2e spec that already tests if the dataframe exist. This is more than enough for `st.write` to test if it works with `pyspark` dataframes.